spec/facets/TagDatasetFacet.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema" : "https://json-schema.org/draft/2020-12/schema",
+  "$id" : "https://openlineage.io/spec/facets/1-0-0/TagDatasetFacet.json",
+  "$defs" : {
+    "TagDatasetFacet" : {
+      "allOf" : [ {
+        "$ref" : "https://openlineage.io/spec/1-0-2/OpenLineage.json#/$defs/DatasetFacet"
+      }, {
+        "type" : "object",
+        "properties" : {
+          "tags" : {
+            "description" : "Tags specified on this table",
+            "type" : "array",
+            "items" : {
+              "type" : "object",
+              "properties" : {
+                "key" : {
+                  "description" : "The name of this tag.",
+                  "type" : "string",
+                  "example" : "key1"
+                },
+                "value" : {
+                  "description" : "The value of this tag.",
+                  "type" : "string",
+                  "example" : "VARCHAR|INT|..."
```

Comment:
Example value suggests as if this was something similar to data type.

Reply:
Fixed.